handbook/working_and_growing_here.md

```diff
@@ -403,20 +579,95 @@ Conscientious objection applies when a high percentage of your time is directed
 
 🗣️ **If your work conflicts with your deeply held beliefs, talk to your manager.**
 
-If you are assigned to a project or long-term relationship with a client that violates your deeply held beliefs, communicate to your manger that you want to be reassigned to a different project or client. Your manager must work in good faith to accommodate the request. If your work is going to make you miserable and hurt your soul, the whole team will suffer— it’s in everyone’s best interest to make a change. [Again, from Tristan](https://docs.google.com/document/d/12nAw4podecG2JWNcslAMlakujmATzXdgkCbfwLPR0sI/view#): 
+If you are assigned to a project or long-term relationship with a client that violates your deeply held beliefs, communicate to your manager that you want to be reassigned to a different project or client. Your manager must work in good faith to accommodate the request. If your work is going to make you miserable and hurt your soul, the whole team will suffer— it’s in everyone’s best interest to make a change. [Again, from Tristan](https://docs.google.com/document/d/12nAw4podecG2JWNcslAMlakujmATzXdgkCbfwLPR0sI/view#): 
 
 “This is not their direct report ‘not being a team player’—rather, it is their direct report optimizing for the long-term. Managers who fail to accommodate such requests risk having high-turnover teams, which will limit their own ability to be successful.”
 
-If you need additional support working with your manager to figure out an accommodation, you can reach out to your People Business Partner, @Meg Colon (Head of People), or @Morgan Russell (DEI Coordinator). 
+If you need additional support working with your manager to figure out an accommodation, you can reach out to your [People Business Partner](https://www.notion.so/dbtlabs/Find-your-PBP-f54e1de2eefa413bad7eb251a3a7c19f?pvs=4).
 
 🛠 **We can help find other solutions when conscientious objection doesn’t solve the problem**
 
 If work that conflicts with your deeply held beliefs is a regular and predictable part of your role rather than an exception, and your right to conscientious objection is impeding the team’s overall success, the best long-term solution may be to look for a role that’s a better fit. You can work with your manager and your People Business Partner to move towards a different role within the company that is more aligned with your personal values. 
 
 💡 You can see some of the discussion that led to this policy in [Working Document: Conscientious Objection](https://www.notion.so/Working-Document-Conscientious-Objection-31977f6aa61b447fabc4f5ac6a20f02f).
 
-## [Employee Referral Program](https://www.notion.so/dbtlabs/How-to-make-a-referral-3d44f9f6b52e49ba8730a3bb48ed2201)
+## Diversity, Equity and Inclusion
```

Comment:
This is duplicate - one should be removed.

Reply:
I'm not seeing the duplicate. Are you referencing the notion link because it's in header (and will be removed)?